src/Otiport.API/Controllers/AddressInformationsController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Otiport.API.Contract.Request.Common;
+using Otiport.API.Contract.Request.Users;
+using Otiport.API.Contract.Response.Common;
+using Otiport.API.Helpers;
+using Otiport.API.Services;
+
+namespace Otiport.API.Controllers
+{
+    [Route("address")]
+    public class AddressInformationsController : ApiControllerBase
+    {
+        private readonly IAddressInformationService _addressInformationService;
+
+        public AddressInformationsController(IAddressInformationService addressInformationService)
+        {
+            _addressInformationService = addressInformationService;
+        }
+        [HttpPost, Route("getcountries")]
```

Comment:
Bu POST olmamalı. REST standartlarına uygun değil. Listeleme işlemi yapıyorsun. Body ihtiyacı yok. Aynı zamanda Route içerisinde get demeye gerek yok zaten Http Method get olması gerektiği için oradan anlaşılır. Sadece "countries" yeterli